surveys/2021-annual-survey/questions.md

```diff
@@ -204,6 +204,27 @@ Select all that apply:
 >
 > This question has led to surprising answers in the past that could help the lang team better prioritize features that make interop easier.
 
+### How much do you agree with the following statements
+
+Statements:
+
+- Rust provides a real benefit over other programming languages
+- Rust is complicated to use
```

Comment:
Maybe also have 'Rust is difficult to learn'?

Reply:
Yea we'll have to watch out for survivorship bias on all of these though.